lib/beacon/template/json_transformer.ex

```diff
@@ -0,0 +1,128 @@
+defmodule Beacon.Template.HEEx.JsonTransformer do
+  # Transforms the AST returned by Beacon.Template.HEEx.Tokenizer.parse into a
```

Comment:
You can transform this comment block into a proper `@moduledoc`

Reply:
done